admin/app/model/deploys/Teamcity.scala

```diff
@@ -79,10 +81,10 @@ class TeamcityService(httpClient: HttpLike) extends ExecutionContexts {
       }
   }
 
-  def getBuilds(project: String, count: Int = 10): Future[Seq[TeamCityBuild]] = {
+  def getBuilds(project: String, branch: String = "master", count: Int = 10): Future[Seq[TeamCityBuild]] = {
     GET[TeamCityBuilds](
       path = "builds",
-      queryString = Map("locator" -> s"buildType:(id:$project),count:$count") + ("fields" -> s"build($buildFields)")
+      queryString = Map("locator" -> s"buildType:(id:$project),branch:$branch,count:$count") + ("fields" -> s"build($buildFields)")
```

Comment:
It looks like if you don't specify a branch value, Teamcity is using the default branch (which is master in our case). So in theory those changes above are not necessary but they make things explicit 💃